### PR TITLE
Don't fail make clean if there is no modest.pc

### PR DESCRIPTION
**Pull request by mody**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -181,7 +181,7 @@ clean: $(MODEST_BUILD_MODULES_TARGET_CLEAN)
 	rm -f $(call MODEST_LIBRARY_WITH_VERSION) && rm -f $(call MODEST_LIBRARY_STATIC)
 	rm -rf $(TEST_DIR_BASE)
 	$(call MODEST_BUILD_CLEAN_AFTER)
-	rm $(MODEST_PKG_CONFIG_FILE)
+	-rm $(MODEST_PKG_CONFIG_FILE)
```
Comment (lexborisov) [inline Makefile]: -rm?

Comment (mody) [reply Makefile]: Meaning don't fail if it's not there.

Comment (mody) [reply Makefile]: I'm not fan of using `rm -f`

Comment (lexborisov) [reply Makefile]: Perhaps the above value also needs to lead to a common style?
```sh
 	-rm $(call MODEST_LIBRARY_WITH_VERSION)
        -rm $(call MODEST_LIBRARY_STATIC)
 	-rm -r $(TEST_DIR_BASE)
```

Comment (mody) [reply Makefile]: Ok, I'll fix them too

**Comment by lexborisov** [comment]
Thanks!

**Pull request merged**